users/tests/test_forms.py

```diff
@@ -50,16 +50,22 @@ def test_duplicate_username(self):
         self.assertIn('username', form.errors)
 
     def test_duplicate_email(self):
+        from django.http import HttpRequest
```

Comment:
Did you mean to use the `RequestFactory` above?

Reply:
yep, thank you 😬